RealmTasks Apple/RealmTasks iOS/CellPresenter.swift

```diff
@@ -106,26 +108,26 @@ class CellPresenter<Item: Object where Item: CellPresentable> {
 
         tableView.contentInset.bottom = 54
         viewController.setTopConstraintTo(constant: 0)
-        UIView.animateWithDuration(0.3) {
+        UIView.animate(withDuration: 0.3) {
             for cell in tableView.visibleCells where cell !== editingCell {
                 cell.alpha = 1
             }
             tableView.superview?.layoutSubviews()
         }
 
         let item = editingCell.item
-        if item.text.isEmpty {
-            let indexPath = NSIndexPath(forRow: items.indexOf(item)!, inSection: 0)
-            item.realm!.delete(item)
-            viewController.tableView.deleteRowsAtIndexPaths([indexPath], withRowAnimation: .Top)
+        if (item?.text.isEmpty)! {
```

Comment:
this is a really funny way to unwrap an optional. `item!.text.isEmpty`. Although I think the problem should actually be solved at the source, which is that `editingCell.item` shouldn't be optional.